libs/core/src/lib/popover/popover-body/popover-body.component.html

```diff
@@ -1,5 +1,5 @@
 <div
-    class="fd-popover__body fd-popover__body--static"
+    class="fd-popover__body"
```

Comment:
<img width="330" alt="Screenshot 2023-05-29 at 18 21 22" src="https://github.com/SAP/fundamental-ngx/assets/28543391/197cf9fd-0c02-4746-97a6-05fd7c597e61">
Offset is lost now